lib/trento/users.ex

```diff
@@ -257,10 +258,20 @@ defmodule Trento.Users do
     Map.put(attrs, :analytics_enabled_at, DateTime.utc_now())
   end
 
-  defp maybe_enable_analytics(attrs) do
+  defp maybe_enable_analytics(%{analytics_enabled: false} = attrs) do
     Map.put(attrs, :analytics_enabled_at, nil)
   end
 
+  # The value for 'analytics_eula_accepted' can only be set
```

Comment:
@jagabomb the comment needs to go above the next `maybe_accept_analytics_eula`. Not here